packages/react/src/SubNav/SubNav.test.tsx

```diff
@@ -200,4 +203,21 @@ describe('SubNav', () => {
 
     expect(queryByRole('link', {name: 'Copilot feature page one'})).not.toBeInTheDocument()
   })
+
+  it('renders an optional subheading into the document', () => {
+    const expectedText = 'Subheading'
+    const expectedLink = '#subheading'
+    const {getByText} = render(
+      <SubNav>
+        <SubNav.Heading href={headingLink}>{heading}</SubNav.Heading>
+        <SubNav.SubHeading href={expectedLink}>{expectedText}</SubNav.SubHeading>
+        <SubNav.Link href="#">Link</SubNav.Link>
+      </SubNav>,
+    )
+
+    const el = getByText(expectedText)
```

Comment:
Very minor, but the following will also check that the element is a link


```suggestion
    const {getByRole} = render(
      <SubNav>
        <SubNav.Heading href={headingLink}>{heading}</SubNav.Heading>
        <SubNav.SubHeading href={expectedLink}>{expectedText}</SubNav.SubHeading>
        <SubNav.Link href="#">Link</SubNav.Link>
      </SubNav>,
    )

    const el = getByRole('link', {name: expectedText})
```